docker_compose/loki-grafana-structured_metadata_map/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+services:
+  fluentbit:
+    image: fluent/fluent-bit:latest
+    depends_on:
+      - loki
+    container_name: fluentbit
+    command: /fluent-bit/bin/fluent-bit -c /etc/fluent-bit_loki_out-structured_metadata_map.yaml
+    ports:
+      - 2021:2021
+    networks:
+      - loki-network
+    volumes:
+      - ./config/fluent-bit_loki_out-structured_metadata_map.yaml:/etc/fluent-bit_loki_out-structured_metadata_map.yaml
```

Comment:
If you mount this to the default location then you can also remove the `command` override - although that only is true currently if it is the legacy TOML format config.

e.g. this would mean no need to have a `command`:
```yaml
      - ./config/fluent-bit_loki_out-structured_metadata_map.conf:/fluent-bit/etc/fluent-bit.conf
```

Reply:
No, this is fine and yeah we will be moving to YAML by default for 4.0.